cmd/nrt/config_split/config.toml

```diff
@@ -5,11 +5,13 @@ TrimColAfterSplit = true # (bool), if true, designated Trim Columns apply to Spl
 WalkSubFolders = true # (bool), if true, splitter process every file including the file in sub-folders; otherwise, ignores sub-folder files.
 
 [Trim]
-Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.      
+Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.                    
 Enabled = true # (bool), turn on/off Trim function.
-OutFolder = "./out/" # (string), in which folder trimmed csv files should be output.
+OutFolder = "./out_trim/" # (string), in which folder trimmed csv files should be output.
 
 [Split]
 Enabled = true # (bool), turn on/off Split function.
+IgnoreFolder = "./ignore/" # (string), If not empty string, dump unsplittable csv file into this folder. If empty, ignore unsplittable csv files.
 OutFolder = "./out_split/" # (string), in which folder split results should be output.
-Schema = ["School", "YrLevel"] # (string array), header sequence for splitting. Each header creates its split category folder.
+Schema = ["School", "YrLevel"] # (string array), header sequence for splitting. Each header creates its split category folder.            
```

Comment:
Request we add "Domain" back in as an option, so that users are not having to guess what the available options are.